README.md

```diff
@@ -178,21 +178,32 @@ This screen can be closed (by tapping the `X` icon) to access other terminal fea
 
 #### PayRequest - Payment Request 
 ```python
+class PaymentMethodType(Enum):
+    CARD = 0
+    ALIPAY = 1
+    WECHAT = 2
+
+class PaymentMethod:
+    payment_method_type: PaymentMethodType
+    activate_qr_code_scanner: bool
+    
 @dataclass
 class PayRequest:
     store_id: str
     terminal_id: str
     amount: float
     show_tips: bool
+    payment_method: PaymentMethod
 ```
 
 Request parameters:
 - `store_id` - the ID of your assigned store
 - `terminal_id` - the serial number of the terminal that will process the payment request. This number is returned by the [list of terminals request](#get-list-of-terminals), or can be found on the back label of the hardware.
 - `amount` - amount as a 2.dp decimal number, such as `"1.00"`
 - `show_tips` - (optional) whether to show (true) or hide (false) the tip options. Default is (false)
-
-
+- `payment_method` - (optional) Settings to enable going straight to QR scanning
+  - `payment_method_type` - Payment method type: ALIPAY, WECHAT
```

Comment:
Should also mention `CARD` here

Reply:
Since CARD is the default value that we handle in kp-core, I didn't think it would add value to the documentation

Reply:
If they always provide the parameter we should specify the default value